src/main/java/org/semux/consensus/SemuxBft.java

```diff
@@ -742,13 +742,13 @@ protected Block proposeBlock() {
         long t1 = TimeUtil.currentTimeMillis();
 
         // fetch pending transactions
-        final List<PendingManager.PendingTransaction> pending = pendingMgr
+        final List<PendingManager.EvaluatedTransaction> pending = pendingMgr
```

Comment:
For VM calls, I'm not sure that 'evaluatedTransaction' really describes their state at this point.  They've been checked for only the most minimal of validation, since we cannot do the same early validation as we do as basic transaction types.